README_src.adoc

```diff
@@ -99,6 +118,10 @@ By default, your ENV variables will be loaded at the root of the context data. F
 
 While the syntax is a little more verbose, paired with `--fail-on-collision`, this option allows ensuring that nothing happens in your back.
 
+=== External files
```

Comment:
I could not push to your branch so you will have to update the mardown. This is just about running a single command.

The easiest **if you have the tools** is to run `just md`. 

Alternatively, you may simply apply the following patch (assuming you do not modify the .adoc further):
```
diff --git a/README.md b/README.md
index fb0caea..e580470 100644
--- a/README.md
+++ b/README.md
@@ -75,6 +75,25 @@ The **tera** engine allows way more than the simple replacements shown above. Yo
 
     cargo install --git https://github.com/chevdor/tera-cli
 
+## Execute from container
+
+### Build container image
+
+    docker build --rm --tag 'tera/cli' .
+
+### Execute `tera` from container
+
+Check tera help
+
+    docker run --interactive --tty --rm tera/cli --help
+
+Parse a template
+
+    docker run --interactive --tty --rm \
+        --volume="$(pwd):/opt" \
+        --env=FOO=BAR \
+        tera/cli --template templates/env-debug.txt --env-only --env-key env
+
 ## What can I do with that anyway ?
 
 Well…​ if you have **data** and you want to format them, this tool will likely be a great companion.
@@ -125,6 +144,10 @@ By default, your ENV variables will be loaded at the root of the context data. F
 
 While the syntax is a little more verbose, paired with `--fail-on-collision`, this option allows ensuring that nothing happens in your back.
 
+### External files
+
+Using the `--include` flag, the command will scan recursively for files that could be [included](https://tera.netlify.app/docs/#include), used as [macros](https://tera.netlify.app/docs/#macros) or for [inheritance](https://tera.netlify.app/docs/#inheritance). By default, it will scan the folder where the main template is located, unless the `--include-path` option is given.
+
 ### Output
 
 By default,
 ```